addon/templates/components/frost-fixed-table.hbs

```diff
@@ -31,8 +31,10 @@
       hook=(concat hookPrefix '-header-left')
       hookQualifiers=hookQualifiers
       tagName='div'
+      isSelectable=_isSelectable
```

Comment:
Not for sorting, this is how the header knows to add in a column in header that corresponds to the checkbox column